test/test_interpolator.py

```diff
@@ -477,7 +480,8 @@ def test_3d_basic():
     ys = np.array([0, 1])
     zs = np.array([0, 1])
 
-    values = (lambda x, y, z: x + y + z)(*np.meshgrid(xs, ys, zs, indexing="ij", sparse=True))
+    values = (lambda x, y, z: x + y + z)(*
+                                         np.meshgrid(xs, ys, zs, indexing="ij", sparse=True))
```

Comment:
Not sure about this one, I'd probably go with
```suggestion
    values = (lambda x, y, z: x + y + z)(
        *np.meshgrid(xs, ys, zs, indexing="ij", sparse=True)
    )
```